changelog-2022.07.01-6.patch

```diff
@@ -1,26 +0,0 @@
-diff --git a/qt/res/app.organicmaps.desktop.metainfo.xml b/qt/res/app.organicmaps.desktop.metainfo.xml
-index a8df72cd76..4a1802dbda 100644
---- a/qt/res/app.organicmaps.desktop.metainfo.xml
-+++ b/qt/res/app.organicmaps.desktop.metainfo.xml
-@@ -107,6 +107,21 @@
-   </screenshots>
- 
-   <releases>
-+   <release version="2022.07.01-6" date="2022-07-01">
```

Comment:
was this changelog entry upstreamed?

Reply:
Yes: https://github.com/organicmaps/organicmaps/pull/2945